### ISSUE TITLE
Deep Phase amplitude (A) formula mismatch

### ISSUE THREAD

**Issue by Mathux** [sebastianstarke/AI4Animation#93]
Hi Sebastian,

Thanks for releasing your great work :) 

To compute the amplitude, it seems to be a mismatch between the formula in the paper and in the code:
- code:    ``amp = 2 * torch.sqrt(torch.sum(power, dim=dim)) / self.time_range``
- paper: ``A = sqrt(2/N * sum(p_ij))``

I think either the code should: ``amp = torch.sqrt(2 / self.time_range * torch.sum(power, dim=dim))``
or the paper should be: ``A = 2 / N * sqrt(sum(p_ij))``

Best,
Mathis

**Comment by Mathux** [comment]
I just found that the 2/N factor is already defined in pi_j, so the formula is correct!
